src/core/libmaven/mavenparameters.cpp

```diff
@@ -26,7 +26,7 @@ MavenParameters::MavenParameters()
         outputdir = "reports" + string(DIR_SEPARATOR_STR);
 
         writeCSVFlag = false;
-        ionizationMode = -1;
+        ionizationMode = 1;
```

Comment:
Any particular reason for changing the default?

Reply:
@shubhra-agrawal +1 is used as default value at every place.

Reply:
@IpankajI Default was -1 earlier. Why change it? There's a possibility default is assumed to be -1 in other parts of the code. It's a risque move in my opinion. We shouldn't change defaults without a strong reason.

Reply:
@shubhra-agrawal  Previously, It was +1 at few places and -1 at some places. I changed it to +1 everywhere

Reply:
@shubhra-agrawal While uploading samples, it still will auto detect ionization-mode from samples.

Reply:
@shubhra-agrawal While uploading samples, it still will auto detect ionization-mode from samples.